src/scss/bitmovinplayer-ui.scss

```
@@ -60,6 +60,8 @@
   color: $color-primary;
   font-family: $font-family;
   font-size: $font-size;
+  font-weight: $font-weight-medium;
+  line-height: $font-size;
```

Nit: It’s usually better to use a unitless `line-height` instead of tying it to a fixed length. A unitless value means children inherit the ratio, not the computed pixels, so their `line-height` adjusts automatically to their own font-size. That keeps headings, captions, and body text looking balanced without extra overrides.

Using a length (e.g. `line-height: $font-size;`) forces all children to inherit the same fixed value, which often leads to cramped or misaligned text.

We could create a unitless variable in this case: `$line-height-default: 1.2 !default;`


```suggestion
  $line-height-default: 1.2 !default; // into variables.scss
  
  line-height: $line-height-default;
```

---

> It’s usually better to use a unitless line-height instead of tying it to a fixed length.

Do you have any references to this?

---

There are a bunch:

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#number
https://css-tricks.com/almanac/properties/l/line-height/
https://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/

---

[7e6bf4c](https://github.com/bitmovin/bitmovin-player-ui/pull/742/commits/7e6bf4c14d9730bafc1be3e02a1088276f93a82e)